surround-cli/README.md

```diff
@@ -0,0 +1,126 @@
+<p align="center">
```

Comment:
This readme needs to be split into 2, the core explanation of how Surround work should be in the library and the command line instructions should be in the cli part.

Reply:
Also, the main readme one needs to be updated as well

Reply:
Yep, I will update the readme once we agree with the overall structure /content